source/community/reactnative/src/components/containers/column.js

```diff
@@ -201,6 +88,40 @@ export class Column extends React.Component {
 			return null;
 		}
 	}
+	/**
+     * @description This function calculates flex value
+	 * 				based on the column width property from the payload. 
+	 * @returns {flex}
+     */
+
+	 flex = (containerViewStyle) => {
```

Comment:
Column layout is getting affected because of these changes. Refer the column based scenarios in Visualizer.

Reply:
Resolved